src/io/fabric8/Utils.groovy

```diff
@@ -162,7 +161,7 @@ boolean isUseDockerSocket() {
     echo "WARNING: Failed to find the getUseDockerSocketFlag() flag on the PipelineConfiguration object - probably due to the jenkins plugin `kubernetes-pipeline-plugin` version: ${e}"
     e.printStackTrace()
   }
-  return supportsOpenShiftS2I() ? false : true;
+  return supportsOpenShiftS2I() ? false : true
```

Comment:
how about `return !supportsOpenShiftS2I()`

Reply:
@sthaha Yes, that works too. This is just automatic semi colon deletion, but I can do that too.